mp/src/game/shared/momentum/mom_system_gamemode.h

```diff
@@ -51,6 +57,7 @@ class CGameMode_Bhop : public CGameModeBase
     const char* GetStatusString() override { return "Bhopping"; }
     const char* GetDiscordIcon() override { return "mom_icon_bhop"; }
     const char* GetMapPrefix() override { return "bhop_"; }
+    const char* GetGameModeCfg() override { return "exec bhop.cfg"; }
```

Comment:
One nitpick I have is the name of this function. I like every class returning the name of the cfg, not the exec command entirely. Why not make the base class call `CFmtStr("exec %s", GetGameModeCfg());`? You also may want to make the empty one a nullptr, and check for null before calling the `engine->` command. A lot cleaner and more flexible, considering RJ will be overriding the exec function and executing either soldier or demo based on the map.

Reply:
Alright I'll do that